### DISCUSSION TITLE
使用telegram adapter的时候，设置了api_server的地址，但是却出现网络无法连接

### DISCUSSION THREAD

**Discussion by BigCat-byebye** [nonebot/discussions#293] [问答] [answered]
如下图所示，能接受到消息，但是却仍旧会报错网络问题，我不太清楚这个网络错误是 哪里来的
![image](https://user-images.githubusercontent.com/32433624/211024967-ea698c0d-5d14-434f-af35-b6812faf4a1c.png)

telegram adapter配置如下
![image](https://user-images.githubusercontent.com/32433624/211024384-972065f4-fc31-4c4c-ab92-faecb0517f50.png)

**Comment by j1g5awi** [comment, marked as answer]
这个错误很早就存在，但我在 codespace 调试时一直没出现，只能归结于网络环境问题。要不你自己去 site-package 里 `nonebot/adapter/telegram/adapter.py` 调试下？

**Reply by BigCat-byebye** [reply]
好的，我刚才已经发现了，是我的梯子有问题
